core/plugins/org.polarsys.capella.core.projection.scenario/src/org/polarsys/capella/core/projection/scenario/commands/ESToISCommand.java

```diff
@@ -147,8 +165,15 @@ public void run() {
                       .filter(sc -> sc.getName().equals(refScenario.getName())).findFirst();
                   if (match.isPresent()) {
                     interaction.setReferencedScenario(match.get());
+                  } else {
+                    interaction.setReferencedScenario(null);
                   }
                 }
+                if (!new MDCHK_InteractionUse_ReferencedScenario().isValidReference(interaction, scenario,
```

Comment:
can you create one instance of MDCHK_InteractionUse_ReferencedScenario outside the loop.

Reply:
ok